skyuxconfig.json

```diff
@@ -4,8 +4,11 @@
   "compileMode": "aot",
   "a11y": true,
   "app": {
-    "styles": [
-      "@skyux/theme/css/sky.css"
-    ]
+    "styles": []
+  },
+  "host": {
+    "frameOptions": {
+      "none": true
```

Comment:
@Blackbaud-ValKuppuswamy Depending how the charts add-in goes, this may need to be set to `"blackbaud": true